robust_intelligence_ai_firewall/manifest.json

```diff
@@ -0,0 +1,66 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "1d208134-9005-4a79-bbc1-445950d1a5c7",
+  "app_id": "robust-intelligence-ai-firewall",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor AI Firewall results using Datadog",
+    "title": "Robust Intelligence AI Firewall",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Robust Intelligence AI Firewall",
+        "image_url": "images/ai-firewall.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Robust Intelligence AI Firewall Results dashboard",
```

Comment:
Apologies for the back and forth here, but this image will need a 16:9 aspect ratio to display properly. Can you add this image onto a 16:9 background, or take a larger screenshot? Thanks!

Reply:
no worries, updated the image ratio.